src/components/shared/PanelSettingsList.css

```diff
@@ -5,14 +5,15 @@
 .panelSettingsList {
   display: flex;
   flex: 1;
-  align-items: flex-start;
+  align-items: stretch;
+  justify-content: flex-start;
   padding: 0;
   margin: 0;
   list-style: none;
 }
 
 .panelSettingsListItem {
-  display: inline-flex;
```

Comment:
`inline-flex` here is no different then `flex`; the parent element has `display: flex` so it ignores whether this element presents itself to the outside as inline or block.